MSAL/test/automation/tests/interactive/MSALB2CInteractiveTests.m

```diff
@@ -63,6 +63,8 @@ - (void)setUp
 
 - (void)runSharedB2CLoginStartWithTestRequest:(MSIDAutomationTestRequest *)request
 {
+    request.validateAuthority = NO;
```

Comment:
@oldalton Is this a temporary fix? For other MSAL libraries the recommendation is to set ValidateAuthority to false : https://learn.microsoft.com/en-us/azure/active-directory-b2c/b2clogin#msalnet-validateauthority-property Maybe we should add the same for objc?
Looking at the validation logic in [MSID](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/92da98d9640fe344837bcce2f33f797ac9349f6b/IdentityCore/src/validation/MSIDB2CAuthority.m#L86) the logic doesn't look right/outdated, I don't believe authority should contain '/tfp' to be valid

Reply:
As per this [documentation](https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-net-aad-b2c-considerations#authority-for-an-azure-ad-b2c-tenant-and-policy), the authority format for Azure AD B2C is: https://{azureADB2CHostname}/tfp/{tenant}/{policyName}  and this is not deprecated . What is [deprecated](https://azure.microsoft.com/en-gb/updates/b2c-deprecate-msol/) is  login.microsoftonline.com endpoint for accessing B2C tenants. 

So i guess the validation logic is still valid ?